webapp/src/Utils/Utils.php

```diff
@@ -1022,6 +1022,32 @@ public static function generatePassword() : string
         return $rand_str;
     }
 
+    /**
+     * Generate a strong random password of the given length
+     *
+     * @param int $length
+     *
+     * @return string
+     */
+    public static function generateStrongPassword($length = 16) : string
```

Comment:
I'm not sure about the name "generateStrongPassword", because it takes a length parameter and e.g. `generateStrongPassword(3)` isn't very strong.

Reply:
In the beginning I wanted to update generatePassword to add options for length and whether to allow more characters. We can also do that?

I will also add unit tests for this function (or the modified generatePassword) btw.

Reply:
Oh right, I missed that there is an existing function called "generatePassword". In that case it makes sense to call the new one "generateStrongPassword" (or maybe rename the existing function to "generateShortPassword", or combine them as you suggested).

Reply:
Is there are reason why we make the length configurable even?